### PR TITLE
Added lua54

### PR DESCRIPTION
**Pull request by Sampleeeee**
Added lua54 to the resource manifest page

### CODE REVIEW

File: content/docs/scripting-reference/resource-manifest/resource-manifest.md [hunk 1]
```diff
@@ -244,6 +244,14 @@ dependencies {
 }
 ```
 
+### lua54
+
+Enabled Lua 5.4. You can read more about Lua 5.4 at https://www.lua.org/manual/5.4/readme.html
```
Comment (Zeemahh) [inline content/docs/scripting-reference/resource-manifest/resource-manifest.md]: Shouldn't this be "Enables Lua 5.4.", not "Enabled"?

File: content/docs/scripting-reference/resource-manifest/resource-manifest.md [hunk 2]
```diff
@@ -244,6 +244,14 @@ dependencies {
...
+
+```lua
+lua54 true
```
Comment (Zeemahh) [inline content/docs/scripting-reference/resource-manifest/resource-manifest.md:252]: Also, just a small thing for consistency here. Other meta-data items will accept any value and will just show `lua54 'yes'`, for example.

Nothing which breaks functionality or changes anything, but just for consistency.

Comment (Sampleeeee) [reply content/docs/scripting-reference/resource-manifest/resource-manifest.md:252]: Ah ok, I asked in the discord and someone said `true` not `'yes'`

Comment (Zeemahh) [reply content/docs/scripting-reference/resource-manifest/resource-manifest.md:252]: Any value works, but it's just for consistency amongst all the other command documentation on this page.

Comment (Sampleeeee) [reply content/docs/scripting-reference/resource-manifest/resource-manifest.md]: Yep, makes sense. Just changed it.

**Review by Zeemahh — APPROVED** [review]

**Pull request merged**